packages/react-native-oursky/src/CountryList.js

```diff
@@ -171,14 +179,24 @@ export default class CountryList extends React.PureComponent<Props, State> {
       visible,
       backButtonText,
       headerTitle,
+      listEmptyComponent,
 
       onPressBackButton,
     } = this.props;
 
-    const { countryCodes, selectedValue } = this.state;
-    const selectedValueIndex = countryCodes.findIndex(
-      item => item.callingCode === selectedValue
+    const { selectedValue, keyword } = this.state;
+    const renderCountryCodes = countryCodes
+      .filter(item =>
+        keyword
+          ? item.name.includes(keyword) || item.callingCode.includes(keyword)
+          : true
+      )
+      .sort(orderByCallingCodeAndName);
+    const selectedValueIndex = renderCountryCodes.findIndex(
+      item => item.callingCode === this.props.selectedValue
     );
+    // If scrollIndex not 0 then FlatList aren't rendered new item until scroll, but it will re-render old item :|
```

Comment:
Have you tried `removeClippedSubviews={false}`?

Reply:
yes, it didn't work